chess-1/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
you can empty this file and instead all content just put project name and one line description.

Reply:
readme changed.